Programs/_testembed.c

```diff
@@ -237,6 +237,9 @@ static void check_stdio_details(const wchar_t *encoding, const wchar_t *errors)
     if (errors) {
         config_set_string(&config, &config.stdio_errors, errors);
     }
+    // gh-106659: On Windows, don't use _io._WindowsConsoleIO which always
+    // announce UTF-8 for sys.stdin.encoding.
+    config.legacy_windows_stdio = 1;
```

Comment:
Do we need here `#ifdef MS_WINDOWS ...`?

Reply:
Oh, I forgot that this option is only available on Windows.